api/app/Providers/EventServiceProvider.php

```diff
@@ -18,6 +19,9 @@ class EventServiceProvider extends ServiceProvider
         Registered::class => [
             SendEmailVerificationNotification::class,
         ],
+        EventCreated::class => [
+            EventCreatedListener::class
```

Comment:
Cmon, we were discussing this. We are going to create separate listeners for each action (from my examples: `SendEventCreatedNotification`, `AddEventToCalendar`, and so on).

Reply:
@aLLeXUs We can add here another listeners wich will call different actions, but they all will listen one common event

Reply:
@dmitriy-uvin yes, the other listeners will be added, but currently, we have `EventCreatedListener` that only sends the notification, so we should name it properly.

Reply:
@aLLeXUs oh, i understand, sure

Reply:
@dmitriy-uvin just `SendEventCreatedNotification`

Reply:
@aLLeXUs done